test/Microsoft.AspNetCore.Mvc.Core.Test/Microsoft.AspNetCore.Mvc.Core.Test.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
     <PreserveCompilationContext>true</PreserveCompilationContext>
+    <NoWarn>$(NoWarn);CS0618</NoWarn>
```

Comment:
How big a change would it be to update tests (perhaps except one or two explicitly testing old constructors) to avoid what's now `[Obsolete]`?

Reply:
Just suppress CS0618 locally, where you're testing `[Obsolete]` constructors et cetera.

Remove the global suppression from Microsoft.AspNetCore.Mvc.Formatters.Json.Test.csproj too.